lib/ferrum_pdf.rb

```diff
@@ -15,9 +15,22 @@ module FerrumPdf
   mattr_accessor :include_controller_module
   @@include_controller_module = true
 
+  mattr_accessor :configuration
+  @@configuration = {}
```

Comment:
```suggestion
  @@configuration = ActiveSupport::OrderedOptions.new
```

Then you can use `config.headless = true` which is more Rails-y.